### ISSUE TITLE
Issue in running code

### ISSUE THREAD

**Issue by yash13902** [KunalGehlot/Voice-Emotion-Recognition#1]
Hello there, I have cloned this project and am running it, but i am not getting the same accuracy... I have changed only few of the depreciated modules in the imports. I am suspecting there is an error with the number of files being used for training and testing. 
Can you please help me, i will really appreciate it as I have been trying this since quite some time.

**Comment by KunalGehlot** [comment]
Hi, 

It has been years since I have even touched this code. Tensorflow and Keras have since moved forward a lot and would NOT produce the same results with the new versions.

I wish I had logged the library versions, but as for now, we cannot do anything but hit and try older versions of libraries.

**Comment by yash13902** [comment]
Ok, thank you for that. I will use this code as a base for doing the analysis.
Thank you!
